crates/ironrdp-cliprdr/Cargo.toml

```diff
@@ -21,4 +21,8 @@ ironrdp-pdu.workspace = true
 ironrdp-svc.workspace = true
 thiserror.workspace = true
 tracing.workspace = true
-bitflags = "2"
\ No newline at end of file
+bitflags = "2"
+
+[target.'cfg(windows)'.dependencies]
+winapi = { version = "0.3", features = ["winuser", "commctrl", "errhandlingapi", "winerror", "winbase"] }
+tokio = { version = "1", features = ["full"]}
```

Comment:
Enabling all the tokio features in a library is not a good practice. I’m also not convinced that `ironrdp-cliprdr` should depend on `tokio` at all, I’ll think about it and review back a bit later

Reply:
Actually yes, we don't need it at all -- that is a leftover from previous implementation iterations, I introduced the `ClipboardMessageProxy` type for the exact purpose - make an abstraction layer from concrete `mpsc` channel type